rosa_tutorials/rosa-consistent-egress-external-traffic.adoc

```diff
@@ -0,0 +1,419 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-consistent-egress-external-traffic"]
+= Assigning consistent egress IP for external traffic
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-consistent-egress-external-traffic
+
+toc::[]
+
+// Mobb content metadata
+// Brought into ROSA product docs 2023-09-19
+// ---
+// date: '2023-02-28'
+// title: Assign Consistent Egress IP for External Traffic
+// tags: ["OSD", "ROSA", "ARO"]
+// authors:
+//  - 'Dustin Scott'
+// ---
+
+It might be desirable to assign a consistent IP address for traffic that leaves the cluster when configuring items such as security groups or other sorts of security controls which require an IP-based configuration. By default, Kubernetes using the OVN-Kubernetes CNI will assign random IP addresses from a pool which will make configuring security lockdowns unpredictable or unnecessarily open. This guide shows you how to configure a set of predictable IP addresses for egress cluster traffic to meet common security standards and guidance and other potential use cases.
+
+See the link:https://docs.openshift.com/container-platform/4.12/networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn.html[OpenShift documentation on this topic] for more information.
+
+== Prerequisites
+
+* ROSA cluster
+* OpenShift CLI (`oc`)
+* ROSA CLI (`rosa`)
+* jq
+
+== Demo
+
+=== Set environment variables
+
+This sets environment variables for the demo so that you do not need to copy/paste in your own.  Be sure to replace the values for your desired values for this step:
```

Comment:
```suggestion
This sets environment variables for the demo so that you do not need to copy/paste in your own. Be sure to replace the values for your desired values for this step:
```